plugins/env/app/controllers/environment_variables_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 class EnvironmentVariablesController < ApplicationController
-  before_action :authorize_admin!, except: [:index]
+  before_action :authorize_admin!, except: [:index, :show, :update]
+  before_action :environment_variable, only: [:show]
```

Comment:
`find_environment` would be more consistent with other controllers, should also use it for update (and edit if that exists)